.github/workflows/robotoff.yml

```diff
@@ -74,9 +74,6 @@ jobs:
       - name: Code quality isort
         run: poetry run isort . --check
 
-      - name: Typing check mypy
```

Comment:
We still want the mypy check, just without 'services/ann':

`poetry run mypy .`